### PR TITLE
koin-androidx-workmanager

### PR DESCRIPTION
**Pull request by arnaudgiuliani**
Add integration of WorkManager, following the work of Fabio @fmatosqg

API is simplified. Also as WorkManager setup declaration.

Let's keep it minimalistic & see how it goes.

### CODE REVIEW

File: koin-projects/koin-androidx-workmanager/src/test/java/org/koin/androidx/workmanager/Classes.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//package org.koin.androidx.workmanager
```
Comment (jeziellago) [inline koin-projects/koin-androidx-workmanager/src/test/java/org/koin/androidx/workmanager/Classes.kt:1]: This file could be removed.

**Comment by jeziellago** [comment]
All test code is fully comments. 👀

**Comment by arnaudgiuliani** [comment]
> All test code is fully comments. 👀

yes, there is no need to test the factory. It's just a simple case for Koin here.

**Comment by jeziellago** [comment]
> > All test code is fully comments. 👀
> 
> yes, there is no need to test the factory. It's just a simple case for Koin here.

Sure! 🚀

**Pull request merged**